### PR TITLE
update origin-aggregated-logging to use es6

### PR DESCRIPTION
**Pull request by jcantrill**
Update jobs to publish es6

### CODE REVIEW

File: core-services/image-mirroring/openshift/mapping_origin_4_5 [hunk 1]
```diff
@@ -138,10 +139,10 @@ registry.svc.ci.openshift.org/ocp/4.5:local-storage-operator quay.io/openshift/o
 registry.svc.ci.openshift.org/ocp/4.5:local-storage-static-provisioner quay.io/openshift/origin-local-storage-static-provisioner:4.5 quay.io/openshift/origin-local-storage-static-provisioner:4.5.0 quay.io/openshift/origin-local-storage-static-provisioner:latest
 registry.svc.ci.openshift.org/ocp/4.5:logging-ci-test-runner quay.io/openshift/origin-logging-ci-test-runner:4.5 quay.io/openshift/origin-logging-ci-test-runner:4.5.0 quay.io/openshift/origin-logging-ci-test-runner:latest
 registry.svc.ci.openshift.org/ocp/4.5:logging-curator5 quay.io/openshift/origin-logging-curator5:4.5 quay.io/openshift/origin-logging-curator5:4.5.0 quay.io/openshift/origin-logging-curator5:latest
```
Comment (ewolinetz) [inline core-services/image-mirroring/openshift/mapping_origin_4_5]: missing curator change

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml [hunk 2]
```diff
@@ -36,11 +36,11 @@ images:
...
-  to: logging-elasticsearch5
+  to: logging-elasticsearch6
 - context_dir: kibana/
   dockerfile_path: Dockerfile.centos7
   from: base_centos
-  to: logging-kibana5
+  to: logging-kibana6
 - context_dir: curator/
   dockerfile_path: Dockerfile.centos7
   from: base_centos
```
Comment (ewolinetz) [inline ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: missing curator change too

Comment (jcantrill) [reply ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: there is no curator6

File: core-services/image-mirroring/openshift/mapping_origin_4_5 (hunk 1 above)
Comment (jcantrill) [reply core-services/image-mirroring/openshift/mapping_origin_4_5]: there is no curator6

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml (hunk 2 above)
Comment (ewolinetz) [reply ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: then we'll need to fix up our oal repo...
https://github.com/openshift/origin-aggregated-logging/blob/feature-es6x/hack/build-images.sh#L240-L256

Comment (jcantrill) [reply ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: https://github.com/openshift/origin-aggregated-logging/pull/1835/commits/523dd966979babcb950947fc8d0d17cdce345091

**Pull request merged**